bitnami/thanos/README.md

```diff
@@ -0,0 +1,559 @@
+# Thanos
+
+[Thanos](https://thanos.io/) is a highly available metrics system that can be added on top of existing Prometheus deployments, providing a global query view across all Prometheus installations.
+
+## TL;DR;
+
+For Helm 2:
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm install --name thanos --namespace thanos bitnami/thanos
+```
+
+For Helm 3:
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+kubectl create namespace thanos
+helm install thanos --namespace thanos bitnami/thanos
+```
```

Comment:
Please use the common way to maintain the same structure than other charts:

> ## TL;DR;
> 
> ```console
> $ helm repo add bitnami https://charts.bitnami.com/bitnami
> $ helm install my-release bitnami/thanos
> ```

please, note the `helm repo add` URL without ending in `/bitnami` and just the Helm 3 approach.